cpplite/cpplite.debugger/src/org/netbeans/modules/cpplite/debugger/CPPLiteDebuggerConfig.java

```diff
@@ -21,6 +21,7 @@
 
 import java.io.File;
 import java.util.List;
+import java.util.Optional;
```

Comment:
`Optional`? I don't like that class much.

Reply:
I can change it to `Long` with `null` allowed, not a problem.

Reply:
I've removed the use of `Optional`.